experimental/framework/fluid-static/README.md

```diff
@@ -87,22 +87,22 @@ const pair1 = initialObjects["pair1"];
 
 ## Using dynamic objects
 
-LoadableObjects can also be created dynamically during runtime. Dynamic object types need to be defined in the  `dynamicObjectTypes` property of the ContainerConfig.
+LoadableObjects can also be created dynamically during runtime. Dynamic object types need to be defined in the  `dynamicObjectTypes` property of the ContainerSchema.
 
 The Container has a `create` method that will create a new instance of the provided type. This instance will be local to the user until attached to another LoadableObject. Dynamic objects created this way should be stored in initialObjects, which are attached when the Container is created. When storing a LoadableObject you must store a reference to the object and not the object itself. To do this use the `handle` property on the LoadableObject.
 
 Dynamic objects are loaded on-demand to optimize for data virtualization. To get the LoadableObject, first get the stored handle then resolve that handle.
 
 ```javascript
-const config = {
+const schema = {
     name: "my-container",
     initialObjects: {
         map1: SharedMap,
     },
     dynamicObjectTypes: [ KeyValueDataObject ],
 }
 
-const container = await Fluid.getContainer("_unique-id_", config);
+const container = await Fluid.getContainer("_unique-id_", schema);
```

Comment:
Nit same with dynamic objects.

Reply:
Removed from here, and it is present in the TinyliciousClient readme